src/sentry/tasks/unmerge.py

```diff
@@ -0,0 +1,468 @@
+from __future__ import absolute_import
+
+import logging
+from collections import defaultdict
+
+from django.db.models import F
+
+from sentry.app import tsdb
+from sentry.constants import DEFAULT_LOGGER_NAME, LOG_LEVELS_MAP
+from sentry.event_manager import ScoreClause, generate_culprit, get_hashes_for_event, md5_from_hash
+from sentry.models import Environment, Event, EventMapping, EventTag, EventUser, Group, GroupHash, GroupRelease, GroupTagKey, GroupTagValue, Project, Release, UserReport
+
+
+def merge_mappings(values):
+    result = {}
+    for value in values:
+        result.update(value)
+    return result
+
+
+initial_fields = {
+    'culprit': lambda event: generate_culprit(
+        event.data,
+        event.platform,
+    ),
+    'data': lambda event: {
+        'last_received': event.data.get('received') or float(event.datetime.strftime('%s')),
+        'type': event.data['type'],
+        'metadata': event.data['metadata'],
+    },
```

Comment:
I'm not 100% sure this data has existed forever, and there's also no explicit schema that requires it, but without it, a lot of code would break … so I'm just sort of assuming it exists … ?